### PR TITLE
docs: readme updated

### PR DESCRIPTION
**Pull request by vrx29**
This PR contains Readme for the notes app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,110 @@
...
+![Forks](https://img.shields.io/github/forks/vrx29/gimesto-keep)
+![Stars](https://img.shields.io/github/stars/vrx29/gimesto-keep)
 
-In the project directory, you can run:
+</div>
+ 
+ ---
 
-### `npm start`
+Gimesto keep is a note taking app. In this you can store your notes and edit them. There is an option to archive your notes and unarchive as well. If you delete a note it will be moved to trash/deleted page from ther you can either restore your notes or delete them permanently.
```
Comment (ApurvaSawant11) [inline README.md:15]: Typo :   page from **there** you can either restore

**Review by ApurvaSawant11 — APPROVED** [review]
Readme looks good!

**Review by anik31 — APPROVED** [review]
Readme looks nice. Good that you've added favicon also. You can also add a demo in the readme.

File: README.md (hunk 1 above)
Comment (vrx29) [reply README.md:15]: fixed

**Pull request merged**